src/test/java/Homework17.java

```diff
@@ -0,0 +1,23 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        navigateToPage();
+        provideEmail("merry.lordlie@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        searchSong("Pluto");
+        clickViewAll();
+        clickFirstSong();
+        clickAddTo();
+        selectMyPlaylist();
+        verifyAddedSongMessage();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**